Classes/Model/Download.cs

```diff
@@ -700,11 +702,16 @@ public static int Compare(int epid1, int epid2)
                         case DownloadCols.Duration:
                             orderBy = "duration" + (sortAsc ? string.Empty : " desc");
                             break;
+                        case DownloadCols.ProgrammeName:
+                            // OrderBy Programme.Name
+                            sqlCmd = "select downloads.epid from downloads, episodes, programmes where downloads.epid=episodes.epid and episodes.progid=programmes.progid order by ";
+                            orderBy = "programmes.name" + (sortAsc ? string.Empty : " desc");
+                            break;
                         default:
                             throw new InvalidDataException("Invalid column: " + sortBy.ToString());
                     }
 
-                    using (SQLiteCommand command = new SQLiteCommand("select downloads.epid from downloads, episodes where downloads.epid=episodes.epid order by " + orderBy, FetchDbConn()))
+                    using (SQLiteCommand command = new SQLiteCommand(sqlCmd + orderBy, FetchDbConn()))
```

Comment:
ok, that makes. made this change but also needed to update the orderBy statement for EpisodeName to remove teh ambiguity of which 'name' column to order by.